### PR TITLE
docs review

### PR DESCRIPTION
**Pull request by wrype**
- k8s-admission-webhook: add webhook.yaml file link
- update redb-role definition
  - according to https://github.com/RedisLabs/redis-enterprise-k8s-docs/tree/master/multi-namespace-redb
- docs review

**The changes effect 6.4-build and master branch.**

### CODE REVIEW

**Comment by wrype** [comment]
@CLAassistant can you assign somebody to review this PR?

**Comment by wrype** [comment]
@dwdougherty hello, can you review this PR?

**Comment by wrype** [comment]
@kaitlynmichael hi, can you review this PR? The changes effect `6.4-build` and `master` branch.

File: content/kubernetes/re-clusters/multi-namespace.md [hunk 1]
```diff
@@ -82,8 +86,8 @@ roleRef:
 Apply the files:
 
 ```sh
-kubectl apply -f role.yaml
-kubectl apply -f role_binding.yaml
+kubectl apply -f role.yaml -n <managed namespace>
```
Comment (zcahana) [inline content/kubernetes/re-clusters/multi-namespace.md:89]: Although it's trivial for some, I think we typically add a comment such as "replace <managed namespace> with the namespace in which you wish to create an REDB", @kaitlynmichael wdyt?

**Review by zcahana — COMMENTED** [review]
@wrype thank you for these helpful corrections.
I've left several minor notes, please take a look.

File: content/embeds/k8s-admission-webhook-cert.md [hunk 2]
```diff
@@ -11,13 +11,15 @@
...
+2. Save the certificate to a local environment variable.
 
     ```sh
     CERT=`kubectl get secret admission-tls -o jsonpath='{.data.cert}'`
     ```
 
-1. Create a patch file for the Kubernetes validating webhook, replacing `<namespace>` with the namespace where the REC was installed.
+3. Create a patch file for the Kubernetes validating webhook, replacing `<namespace>` with the namespace where the REC was installed.
+
+  The `webhook.yaml` example is in [redis-enterprise-k8s-docs/admissiion](https://github.com/RedisLabs/redis-enterprise-k8s-docs/tree/master/admission)
```
[the remark this answers is not in the document]
Comment (wrype) [reply content/embeds/k8s-admission-webhook-cert.md]: done

File: content/embeds/k8s-admission-webhook-cert.md [hunk 3]
```diff
@@ -17,13 +17,17 @@
     CERT=`kubectl get secret admission-tls -o jsonpath='{.data.cert}'`
     ```
 
-3. Create a patch file for the Kubernetes validating webhook, replacing `<namespace>` with the namespace where the REC was installed.
-
-  The `webhook.yaml` example is in [redis-enterprise-k8s-docs/admissiion](https://github.com/RedisLabs/redis-enterprise-k8s-docs/tree/master/admission)
+3. Create a Kubernetes validating webhook, replacing `<namespace>` with the namespace where the REC was installed.
+   
+    > The `webhook.yaml` template can be found in [redis-enterprise-k8s-docs/admission](https://github.com/RedisLabs/redis-enterprise-k8s-docs/tree/master/admission)
```
Comment (zcahana) [inline content/embeds/k8s-admission-webhook-cert.md:22]: I'm not sure how `>` lines are rendered in this website and if it's something we'd actually want for this sentence. Maybe it's best to just omit the `>`.

Comment (wrype) [reply content/embeds/k8s-admission-webhook-cert.md]: done

**Review by zcahana — APPROVED** [review]
Looks good from my perspective but let's have @kaitlynmichael do the final ack.

File: content/kubernetes/re-clusters/multi-namespace.md (hunk 1 above)
Comment (kaitlynmichael) [inline content/kubernetes/re-clusters/multi-namespace.md:89]:
```suggestion
kubectl apply -f role.yaml -n <managed-namespace>
```

File: content/kubernetes/re-clusters/multi-namespace.md [hunk 4]
```diff
@@ -82,8 +86,8 @@ roleRef:
...
+kubectl apply -f role_binding.yaml -n <managed namespace>
```
Comment (kaitlynmichael) [inline content/kubernetes/re-clusters/multi-namespace.md:90]:
```suggestion
kubectl apply -f role_binding.yaml -n <managed-namespace>
```

File: content/kubernetes/re-clusters/multi-namespace.md [hunk 5]
```diff
@@ -82,8 +86,8 @@ roleRef:
 Apply the files:
```
Comment (kaitlynmichael) [inline content/kubernetes/re-clusters/multi-namespace.md:86]:
```suggestion
Apply the files, replacing `<managed-namespace>` with your own values:
```

**Review by kaitlynmichael — CHANGES_REQUESTED** [review]
I suggested a few changes and then this is ready to merge.

File: content/kubernetes/re-clusters/multi-namespace.md (hunk 1 above)
Comment (wrype) [reply content/kubernetes/re-clusters/multi-namespace.md]: done

File: content/kubernetes/re-clusters/multi-namespace.md (hunk 4 above)
Comment (wrype) [reply content/kubernetes/re-clusters/multi-namespace.md]: done

File: content/kubernetes/re-clusters/multi-namespace.md (hunk 5 above)
Comment (wrype) [reply content/kubernetes/re-clusters/multi-namespace.md]: done

**Review by zcahana — APPROVED** [review]

**Comment by wrype** [comment]
@zcahana hi, could you cherry pick this PR to the branch `6.4-build` as well?

**Pull request merged**